chartfx-chart/src/main/java/io/fair_acc/chartfx/Chart.java

```diff
@@ -928,10 +927,10 @@ private void ensureJavaFxPulse() {
         return CSS.getCssMetaData();
     }
 
-    protected static Group createChildGroup() {
-        final Group group = new Group();
+    protected static Pane createChildGroup() {
```

Comment:
maybe call it `createUnmanagedPane()` or move it to some util class

Reply:
Will do.